src/openfl/_internal/renderer/opengl/batcher/QuadTextureData.hx

```diff
@@ -7,6 +7,9 @@ class QuadTextureData {
 
 	/** Texture coordinates (0x0-1x1 for full texture, some region for atlas sub-textures) **/
 	public var uvs(default,null):Float32Array;
+	
+	//Is the texture with premutliplied alpha or not
+	public var premultipliedAlpha(default,null):Bool;
```

Comment:
Please use a consistent style for field comments

Reply:
You wrote "premutliplied" here and above you have "vPremultipledAlpha" and "aPremultipledAlpha" typos ;)